index.css

```diff
@@ -342,14 +349,23 @@ html {
 }
 
 .perks-col {
-  flex: 1 1 0;
+  flex: 1 1 50%;
+}
+
+.perks-col.text {
   padding: 20px;
 }
 
 .perks-col.pic {
   background: #ccc;
 }
 
+.perks-col.pic img {
+  object-fit: cover;
```

Comment:
wow this is cool. never seen it

Reply:
Yeah I know right! Learnt about it from Wes Bos 👌